qiita_db/metadata_template.py

```diff
@@ -620,23 +620,37 @@ def __setitem__(self, column, value):
         """
         conn_handler = SQLConnectionHandler()
 
-        exists = conn_handler.execute_fetchone("""
+        # try dynamic tables
+        exists_dynamic = conn_handler.execute_fetchone("""
             SELECT column_name
             FROM information_schema.columns
             WHERE table_name='{0}'
                 AND table_schema='qiita'
                 AND column_name='{1}'""".format(self._dynamic_table, column))
-
-        if exists is None:
+        # try required_sample_info
+        exists_required = conn_handler.execute_fetchone("""
```

Comment:
I can use that one but note that `get_table_cols` doesn't use `AND table_schema='qiita'`, any reason for this?